index.html

```diff
@@ -724,6 +724,18 @@ <h4 id="turning-off-protections">Tuning off protections</h4>
         useful during development.
       </p>
 
+      <h3 id="dsd">Declarative Shadow DOM</h3>
+
+      <p>
+        Support for parsing fragments of HTML which include declarative shadow DOM (DSD) templates via <code>Document.parseHTMLUnsafe</code> <a href="https://developer.mozilla.org/en-US/docs/Web/API/Document/parseHTMLUnsafe_static">recently entered Baseline 2024 status</a>, so <code>&lt;i-html></code> will use that method of parsing if supported. Older browsers would need a polyfill to handle those cases, otherwise, <code>&lt;i-html></code> will fall back on <code>new DOMParser().parseFromString</code> which does not support DSD.
```

Comment:
It would be useful to explicitly call out the nomenclature around `parseHTMLUnsafe` to avoid FUD (it's not "that" unsafe, and merely means no sanitization is taking place during parse - but we sanitize after parse). Also perhaps vow to include `parseHTML` when in ships (AIUI no one ships `parseHMTL` yet, right?).

Reply:
yeah, I thought of that. "we make the unsafe safe!" is a hard sell, but I'll see if I can come up with some wording on that.